src/components/Items/Question/Body.vue

```diff
@@ -6,62 +6,67 @@
         {{ questionText }}
       </p>
     </div>
-    <!-- option container -->
-    <div v-if="isQuestionTypeMCQ" class="flex mx-4 md:mx-6 xl:mx-10">
-      <ul class="w-full">
-        <li class="list-none space-y-1 flex flex-col">
-          <div
-            v-for="(option, optionIndex) in options"
-            :key="optionIndex"
-            :class="[optionBackgroundClass(optionIndex), optionTextClass]"
-          >
-            <!-- each option is defined here -->
-            <!-- adding <label> so that touch input is just
-                  not limited to the radio button -->
-            <label :class="labelClass(option)">
-              <!-- understand the meaning of the keys here:
-               https://www.w3schools.com/tags/att_input_type_radio.asp -->
-              <input
-                type="radio"
-                name="questionOptions"
-                :value="option"
-                class="place-self-center"
-                @click="selectOption(optionIndex)"
-                :checked="isOptionChecked(optionIndex)"
-                :disabled="isAnswerSubmitted || previewMode"
-              />
-              <div
-                v-html="option"
-                class="ml-2 h-full place-self-center leading-tight"
-              ></div>
-            </label>
-          </div>
-        </li>
-      </ul>
-    </div>
-    <!-- subjective question answer -->
-    <div
-      v-if="isQuestionTypeSubjective"
-      class="flex flex-col px-4 md:px-6 xl:px-10 w-full"
-    >
-      <!-- input area for the answer -->
-      <Textarea
-        :placeholder="subjectiveAnswerInputPlaceholder"
-        class="px-2 w-full"
-        v-model:value="subjectiveAnswer"
-        boxStyling="px-4 placeholder-gray-400 bp-420:h-20 sm:h-28 md:h-36 focus:border-gray-200 focus:ring-transparent"
-        :isDisabled="isAnswerSubmitted || previewMode"
-        @keypress="checkCharLimit"
-        :maxHeightLimit="subjectiveBoxHeightLimit"
-      ></Textarea>
-      <!-- character limit -->
+    <div :class="orientationClass">
+      <!-- question image container -->
+      <div :class="questionImageContainerClass" v-if="isQuestionImagePresent">
+        <img :src="questionImageUrl" class="object-contain h-full w-full" />
+      </div>
```

Comment:
the image was going to come above the question text for portrait mode and on the left of the question (and options) in the landscape mode right?